app/models/invenio_rdm_record_converter.rb

```diff
@@ -56,7 +58,8 @@ def record_for_export(generic_file)
     {
       "pids": invennio_pids(generic_file.doi.shift),
       "metadata": invenio_metadata(generic_file),
-      "provenance": invenio_provenance(generic_file.proxy_depositor, generic_file.on_behalf_of)
+      "provenance": invenio_provenance(generic_file.proxy_depositor, generic_file.on_behalf_of),
+      "access": invenio_access(generic_file.visibility)
```

Comment:
Turns out embargo is an optional field which itself contains required fields. Since, per our discussion this morning at standup, there is no embargo date on DigitalHub it is being left out.